AutoRouteManager.php

```diff
@@ -64,22 +71,45 @@ public function __construct(
      */
     public function buildUriContextCollection(UriContextCollection $uriContextCollection)
     {
-        $this->getUriContextsForDocument($uriContextCollection);
+        $this->collectionBuilder->build($uriContextCollection);
```

Comment:
The collection builder creates all of the `UriContext` instances - one context per locale and per "definition".